src/components/Model3DViewer/Model3DViewer.tsx

```diff
@@ -217,7 +218,10 @@ export class Model3DViewer extends React.Component<Model3DViewerProps> {
 
   render() {
     return (
-      <div style={{ width: '100%', height: '100%' }} ref={this.divWrapper} />
+      <div
+        style={{ width: '100%', height: '100%', fontSize: 0 }}
```

Comment:
fontSize?

Reply:
yes, its related to @cognite/3d-viewer `_resizeIfNeeded` function, which is check wrapper dimension and fit canvas to it (if they are different). But after changing of canvas size, dimension of wrapper also grows, so it cause infinite increasing of canvas. If font-size of wrapper is 0, it has the same size as canvas and doesn't trigger `_resizeIfNeeded`